app/src/main/res/values/arrays.xml

```diff
@@ -7,6 +7,7 @@
         <item>Wifi Wixel + BT Wixel</item>
         <item>G4 Share Receiver</item>
         <item>G5 Transmitter</item>
+        <item>LimiTTer</item>
```

Comment:
a few questions:
1) If I understand correctly, http://joernl.github.io/LimiTTer/ does not contain any sw, is that correct?
2) Does the limiter use calibrations? If not we can probably remove them from the menus.

As for the name itself, shouldn't the name that the user is choosing be something like Freestyle?